frontend/src/app/features/work-packages/components/wp-tabs/components/wp-tabs/wp-tabs.component.ts

```diff
@@ -64,6 +66,10 @@ export class WpTabsComponent implements OnInit {
   }
 
   public close():void {
+    const { baseRoute } = this.uiRouterGlobals.current.data as { baseRoute:string }&unknown;
+    if (baseRoute.includes('notifications')) {
+      this.titleService.setFirstPart('OpenProject');
```

Comment:
It might be better to make this an i18n string. I'm not sure if the title is configurable and comes from the backend somewhere.

Reply:
Yeah, definitely! The instance name is not always 'OpenProject' but may be something customised.  I don't know if we have that Information available in frontend though, so you can also use 'Notification center' .